backend/src/types.ts

```diff
@@ -1205,3 +1205,9 @@ export type ResponseStatus = {
   success: boolean;
   error: string;
 };
+
+export enum ServiceAddressType {
+  INTERNAL = 'internal',
+  EXTERNAL_REST = 'rest',
+  EXTERNAL_GRPC = 'grpc',
+}
```

Comment:
Couldn't this and `SERVICE_ADDRESS_ANNOTATIONS` be combined so that the values of each are annotations ?

Reply:
Done